src/components/table/EnhancedTableToolbar.tsx

```diff
@@ -31,8 +32,8 @@ export default function EnhancedTableToolbar(props: EnhancedTableToolbarProps):
       <Typography color='inherit' variant='subtitle1' component='div' className={classes.flexText}>
         {renderNumSelectedText(numSelected)}
       </Typography>
-      {topBarButtons?.map((tbButton) => {
-        return (
+      {topBarButtons?.map((tbButton, index) => (
+        <Tooltip title={tbButton.tooltipTitle} key={index}>
```

Comment:
Should we validate it has a tooltipTitle before creating the tooltip?

Reply:
I started out that way, ([first commit](https://github.com/terraware/web-components/pull/438/commits/3113694fba7690c55fb91fbb390ebd28d21bdc98)). Switched to this simpler approach, it will still render the children with undefined tooltip.